src/requestforattestation/RequestForAttestation.spec.ts

```diff
@@ -115,6 +124,54 @@ describe('RequestForAttestation', () => {
     expect(request.claim.owner).toBeUndefined()
   })
 
+  it('compresses and decompresses the request for attestation object', () => {
+    const legitimationAttestationBob: Attestation = Attestation.fromRequestAndPublicIdentity(
+      legitimationRequest,
+      identityBob
+    )
+    const legitimationBob: AttestedClaim = AttestedClaim.fromRequestAndAttestation(
+      legitimationRequest,
+      legitimationAttestationBob
+    )
+    const reqForAtt: RequestForAttestation = buildRequestForAttestation(
+      identityBob,
+      'ctype',
+      {
+        a: 'a',
+        b: 'b',
+        c: 'c',
+      },
+      [legitimationCharlie, legitimationBob]
+    )
+
+    const sortedCompressedReqForAtt: CompressedRequestForAttestation = [
+      compressClaim(reqForAtt.claim),
```

Comment:
try to avoid using these helper functions

Reply:
I am not sure I can.

Reply:
I _am_ sure you can ;)

Reply:
Ok, I have created this monstrosity :D

Reply:
love it